### PR TITLE
adding simplejson and ckanapi

### PR DESCRIPTION
**Pull request by rsignell-usgs**

### CODE REVIEW

File: ckanapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools
```
Comment (ocefpaf) [inline ckanapi/meta.yaml]: duplicated

File: ckanapi/bld.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```
Comment (ocefpaf) [inline ckanapi/bld.bat]: When a package uses `setuptools` we can add `--single-version-externally-managed --record record.txt` here and below.  That makes the package more "portable."

File: ckanapi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    - docopt
+    - requests
+    - simplejson
+
+  run:
+    - python
+    - setuptools
+    - setuptools
```
Comment (ocefpaf) [inline ckanapi/meta.yaml]: Duplicated again.

File: ckanapi/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools
+    - docopt
+    - requests
+    - simplejson
```
Comment (ocefpaf) [inline ckanapi/meta.yaml]: If you add `setuptools` we can add `--single-version-externally-managed --record record.txt` you can probably remove `docopt`, `requests`, and `simplejson` from the build step.

File: simplejson/build.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (ocefpaf) [inline simplejson/build.sh]: Same comment as abouve about the `--single-version-externally-managed --record record.txt`

File: ckanapi/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    #
+    # - ckanapi = ckanapi:main
+    #
+    # Would create an entry point called ckanapi that calls ckanapi.main()
+
+    - ckanapi=ckanapi.cli.main:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
```
Comment (ocefpaf) [inline ckanapi/meta.yaml]: Uncomment this and change to `0`

File: simplejson/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    # syntax is module:function.  For example
+    #
+    # - simplejson = simplejson:main
+    #
+    # Would create an entry point called simplejson that calls simplejson.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
```
Comment (ocefpaf) [inline simplejson/meta.yaml]: Uncomment this and change to `0`

**Comment by ocefpaf** [comment]
@rsignell-usgs I just reviewed and travis is complaining about the lack of `mock` to run the tests.  Adding that to the run dependencies is not recommended because we won't need `mock`.  I would remove https://github.com/ioos/conda-recipes/pull/288/files#diff-5905a7c04e784b169dc52ffd8dac8a5eR52 instead.

Also, for clarity in future changes, I recommend removing all the "commented out boilerplate" that conda generates when creating the package files.

File: ckanapi/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  md5: 2ab2508df730b0e7d84e13be74adc314
+
+build:
+    number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docopt
```
Comment (ocefpaf) [inline ckanapi/meta.yaml]: These are `run` dependencies and should be below.

PS: In my comment I mentioned `build` and not `run` :stuck_out_tongue_winking_eye:

File: simplejson/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  imports:
+    - simplejson
+    - simplejson.tests
+
+about:
+  home: http://github.com/simplejson/simplejson
+  license: MIT License or Academic Free License (AFL)
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+
+# See
```
Comment (ocefpaf) [inline simplejson/meta.yaml]: Left over boilerplate comments.

**Comment by rsignell-usgs** [comment]
Do things like this work "as is" in conda or do we need to add more stuff to get the "speedups" to work?
https://github.com/simplejson/simplejson/blob/master/setup.py#L89

**Comment by ocefpaf** [comment]
Setuptools should take care of that.  We can add an "import speedup" test to be sure the build works.

**Pull request merged**